models/app.models.js

```diff
@@ -37,4 +37,43 @@ exports.fetchArticles = () =>{
     .then(({rows})=>{
         return rows
     })
+}
+
+exports.fetchCommentsByArtId = (article_id)=>{
+    const articleString = `
+    SELECT * FROM articles
+    WHERE article_id = $1;
+`;
+return db
```

Comment:
You already trying to get an article with that ID from your controller using the original function for that. Is it necessary to get that article again here?

Reply:
done.